SofascoreAcademy/SofascoreAcademy/Attributes/String+Common.swift

```diff
@@ -29,4 +29,5 @@ extension String {
     static let settings_title = NSLocalizedString("Settings", comment: "Settings title")
     static let dismiss_string = NSLocalizedString("Dismiss", comment: "Dismiss label")
     static let event_details_title = NSLocalizedString("Event Details", comment: "Event details title")
+    static let sofaApiUrlString = "https://static-api.sofascore.dev/api/event/"
```

Comment:
Ovo isto nebi trebalo biti tu. U ovom fielu su samo ono što je vidljivo na screenu.